data.sql

```diff
@@ -23,3 +23,25 @@ UPDATE animal SET owners_id = '2' WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animal SET owners_id = '3' WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animal SET owners_id = '4' WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
 UPDATE animal SET owners_id = '5' WHERE name = 'Angemon' OR name = 'Boarmon';
+
+INSERT INTO vets(name,age,date_of_graduation) VALUES('William Tatcher',45,'2000-04-23'),('Maisy Smith',26,'2019-01-17'), ('Stephanie Mendez',64,'1981-05-04'),('Jack Harness',38,'2008-06-08');
+
+INSERT INTO visits(animals_id,vets_id,date_of_visit) VALUES(1,1,'2020-05-24'),(1,4,'2020-07-22'),(2,3,'2021-02-02'),(3,2,'2020-01-05'),(3,2,'2020-03-08'),(3,2,'2020-05-14'),(4,4,'2021-05-04'),(5,3,'2021-02-24'),(6,2,'2019-12-21'),(6,1,'2020-08-10'),(6,2,'2021-04-07'),(7,4,'2019-09-29'),(8,3,'2020-10-03'),(8,3,'2020-11-04'),(9,2,'2019-01-24'),(9,2,'2019-05-15'),(9,2,'2020-02-27'),(9,2,'2020-08-03'),(10,4,'2020-05-24'),(10,1,'2021-01-11');
+
+SELECT visits.animals_id,animal.name FROM visits JOIN vets ON visits.vets_id = vets.id JOIN animal ON animal.id = visits.animals_id WHERE visits.vets_id = vets.id AND vets.name = 'William Tatcher' ORDER BY animals_id DESC LIMIT 1;
+
+SELECT COUNT(DISTINCT animal.name) FROM visits JOIN vets ON visits.vets_id = vets.id JOIN animal ON animal.id = visits.animals_id WHERE visits.vets_id = vets.id AND vets.name = 'Stephanie Mendez';
+
+SELECT * FROM vets FULL JOIN specialization on vets.id = specialization.vets_id FULL JOIN species ON species.id = specialization.vets_id;
+
+SELECT animal.name,visits.date_of_visit  FROM visits JOIN vets ON visits.vets_id = vets.id JOIN animal ON animal.id = visits.animals_id WHERE visits.vets_id = vets.id AND vets.name = 'Stephanie Mendez' AND visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+
```

Comment:
- [ ] It would be nice if this query returns two entries because this query is not giving the desired data 👍 because you have not inserted data correctly which I mentioned above 👍 Kindly check that inline comment about the id 4 and id 3 confusion 🤔

- [ ] After, making the changes it would be very great if you take another screenshot of queried data and replace with the old one in the pull request summary to let the other reviewer know that you have made the change correctly 👍